client/my-sites/promote-post-i2/components/campaign-item/index.tsx

```diff
@@ -140,14 +145,24 @@ export default function CampaignItem( props: Props ) {
 					</div>
 				</div>
 			</td>
-			<td className="campaign-item__user">{ display_name }</td>
-			<td className="campaign-item__status">{ statusBadge }</td>
+			<td className="campaign-item__user">
+				<div>{ display_name }</div>
+			</td>
+			<td className="campaign-item__status">
+				<div>{ statusBadge }</div>
+			</td>
 			<td className="campaign-item__ends">
-				{ getCampaignEndText( moment, campaign.status, campaign.end_date ) }
+				<div>{ getCampaignEndText( moment, campaign.status, campaign.end_date ) }</div>
+			</td>
+			<td className="campaign-item__budget">
+				<div>{ budgetString }</div>
+			</td>
+			<td className="campaign-item__impressions">
+				<div>{ formatNumber( impressions_total ) }</div>
+			</td>
+			<td className="campaign-item__clicks">
+				<div>{ formatNumber( clicks_total ) }</div>
 			</td>
-			<td className="campaign-item__budget">{ budgetString }</td>
-			<td className="campaign-item__impressions">{ formatNumber( impressions_total ) }</td>
-			<td className="campaign-item__clicks">{ formatNumber( clicks_total ) }</td>
 			<td className="campaign-item__action">
 				<Button href={ openCampaignURL } isLink icon={ chevronRight } />
```

Comment:
I don't know why, but when you click on the details link, its forcing a full page reload.

I imagine this happens because both the button and the row click event run simultaneously. We may need to add a stop.propagation in the button onClick event.

Reply:
I fixed both issues..
for the > button I just removed the href property (it gets it either way from the onClick.. so no need to have an extra href there)

for the title .. i just removed the max-width.. i have set width properties for the td.. so it is ok in desktop